### PR TITLE
Add FT Factories + update contracts, txns & config to source

### LINKED ISSUE #5: Update with current HybridCustody version
Source contracts have undergone minor updates, and this scaffold needs to be updated as well.

### PR DESCRIPTION
**Pull request by sisyphusSmiling**
Closes: #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -549,4 +549,25 @@ ecosystem while you can focus on delivering stellar in-app experiences!
...
+### Hosted `CapabilityFactory` & `CapabilityFilter` Implementations
+
+> :information_source: `CapabilityFactory.Manager` implementations and `CapabilityFilter.AllowAllFilter` have been
+> deployed to the accounts below for generalized use cases to make account linking as easy as possible. These
+> generalized implementations likely cover most use cases, but you'll want to weigh the decision to use them according
+> to your risk tolerance and specific scenario.
+
+| Use Case | Testnet Address                                                                       | Mainnet Address                         |
+| -------- | ------------------------------------------------------------------------------------- | --------------------------------------- |
+| NFT Capability Factories       | [0x1055970ee34ef4dc](https://f.dnz.dev/0x1055970ee34ef4dc/storage/CapabilityFactory_0x294e44e1ec6993c6) | [0xee9ff4f07a2d6dad](https://f.dnz.dev/0xee9ff4f07a2d6dad/storage/CapabilityFactory_0xd8a7e05a7ac670c0) |
```
Comment (joshuahannan) [inline README.md:570]: Any reason these were all deployed to different addresses?

File: cadence/contracts/hybrid-custody/CapabilityFilter.cdc [hunk 2]
```diff
@@ -112,6 +120,14 @@ pub contract CapabilityFilter {
...
             }
         }
+
+        /// Removes all types from the mapping of denied types
+        ///
+        pub fun removeAllTypes() {
+            for type in self.allowedTypes.keys {
+                self.removeType(type)
+            }
+        }
```
Comment (joshuahannan) [inline cadence/contracts/hybrid-custody/CapabilityFilter.cdc:130]:
```suggestion
        /// Removes all types from the mapping of allowed types
        ///
        pub fun removeAllAllowedTypes() {
            for type in self.allowedTypes.keys {
                self.removeType(type)
            }
        }
```

File: cadence/contracts/hybrid-custody/CapabilityFilter.cdc [hunk 3]
```diff
@@ -54,6 +54,14 @@ pub contract CapabilityFilter {
             }
         }
 
+        /// Removes all types from the mapping of denied types
+        ///
+        pub fun removeAllTypes() {
```
Comment (joshuahannan) [inline cadence/contracts/hybrid-custody/CapabilityFilter.cdc:59]:
```suggestion
        pub fun removeAllDeniedTypes() {
```

**Review by joshuahannan — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (sisyphusSmiling) [reply README.md:570]: The common Manager defined in `CapabilityFactory` contract has canonical paths for where to find the Manager resource with the assumption of a single Manager per account. Deploying to the same account would create path namespace collisions.

The thought is that it's easier and more intuitive to create boilerplate with address vs path params to select a use case-specific Factory Manager. Does that make sense?

**Pull request merged**